helm/examples/logging/README.md

```diff
@@ -0,0 +1,57 @@
+
+    For example:
+    >kubectl logs myesp-6476c6659b-vqckq
+    >0000CF0F PRG INF 2020-05-12 17:10:34.910     1 10690 "HTTP First Line: GET / HTTP/1.1"
+    >0000CF10 PRG INF 2020-05-12 17:10:34.911     1 10690 "GET /, from 10.240.0.4"
+    >0000CF11 PRG INF 2020-05-12 17:10:34.911     1 10690 “TxSummary[activeReqs=22; rcv=5ms;total=6ms;]"
+ 
+It is important to understand that these logs are ephemeral in nature, and may be lost if the pod is evicted, the container crashes, the node dies, etc. Also, due to the nature of containerized solutions, related logs are likely to originate from various locations and might need to be collected and processed. It is highly recommended to develop a retention and processing strategy based on your needs.
+
+Many tools are available to help create an appropriate solution based on either a do-it-yourself approach, or managed features available from cloud providers.
+
+For the simplest of environments, it might be acceptable to rely on the standard Kubernetes process which forwards all contents of stdout/stderr to file. However, as the complexity of the cluster grows or the importance of retaining the logs' content grows, a cluster-level logging architecture should be employed.
+
+Cluster-level logging for the containerized HPCC Systems cluster can be accomplished by including a logging agent on each node. The task of each of agent is to expose the logs or push them to a log processing backend. Logging agents are generally not provided out of the box, but there are several available such as Elasticsearch and Stackdriver Logging. Various cloud providers offer built-in solutions which automatically harvest all stdout/err streams and provide dynamic storage and powerful analytic tools, and the ability to create custom alerts based on log data.
+
+_Specific logging tool examples can be found in the child folders_
+
+## HPCC Systems application level logging details
+
+As mentioned earlier, the HPCC Systems logs provide a wealth of information which can be used for benchmarking, auditing, debugging, monitoring, etc. The type of information provided in the logs and its format is trivially controlled via standard Helm configuration.
+
+By default, the component logs are not filtered, and contain the following columns:
+    
+    MessageID TargetAudience DateStamp TimeStamp ProcessId ThreadID QuotedLogMessage
+
+The logs can be filterd by TargetAudience, or by Category, and the output columns can be configured. Logging configuration settings can be applied at the global, or component level.
```

Comment:
> filterd

filtered

Reply:
Also looks like rather a long line?

Reply:
Other lines are longer (e.g. line 19),
but they format ok (MD) when viewed in e.g. github, e.g.: https://github.com/hpcc-systems/HPCC-Platform/blob/2bdc2bba6defbc87c7d9611ad027522a9a2b86ef/helm/examples/logging/README.md